agent/command/git.go

```diff
@@ -735,6 +735,7 @@ func (c *gitFetchProject) fetchModuleSource(ctx context.Context,
 		appToken, err := comm.CreateInstallationToken(ctx, td, opts.owner, opts.repo)
 		if err == nil {
 			opts.token = appToken
+			opts.method = cloneMethodAccessToken
```

Comment:
Is this relevant to the ticket? Seems kind of random since this is for git.get_project, not `evergreen fetch`.

Reply:
it's not, I just noticed it and it was just easier to roll it in here than create a separate PR.